blockchains/elrond/assets/EGLDUSDCF-8600f8/info.json

```
@@ -0,0 +1,44 @@
+{
+    "type": "ESDTSFT",
+    "id": "EGLDUSDCF-8600f8",
+    "name": "EGLDUSDCLPStaked",
```

```suggestion
    "name": "EGLD USDC/LP Staked",
```

Would it make more sense to make it this way?

---

Renamed.